run-sidecar

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+CURSORLESS_FORK_REPO="git@github.com:cursorless-everywhere/cursorless.git"
+CURSORLESS_FORK_REV="main"
+CURSORLESS_FORK_NAME="pokey-phillco.cursorless-fork"
+
+CURSORLESS_SIDECAR_REPO="git@github.com:cursorless-everywhere/sidecar.git"
+CURSORLESS_SIDECAR_REV="main"
+CURSORLESS_SIDECAR_NAME="phillco.cursorless-sidecar"
+
+if ! [ -x "$(command -v code)" ]; then
+    echo 'Error: code is not installed.' >&2
+    exit 1
+elif ! [ -x "$(command -v yarn)" ]; then
+    echo 'Error: yarn is not installed.' >&2
+    exit 1
+fi
+
+POSITIONAL_ARGS=()
+
+while test $# -gt 0; do
+    case "$1" in
+        -h | --help)
+            echo "run-sidecar - Provision and run the cursorless sidecar against a vscode extensions directory."
+            echo " "
+            echo "run-sidecar [OPTION]... [EXTENSION_DIR]"
+            echo " "
+            echo "options:"
+            echo "-h, --help                    show brief help"
+            echo "-c, --cursorless-revision     use a specific revision of the cursorless fork"
+            echo "-s, --sidecar-revision        use a specific revision of the cursorless sidecar"
+            exit 0
+            ;;
+        -c | --cursorless-revision)
+            CURSORLESS_FORK_REV="$2"
+            shift
+            shift
+            ;;
+        -s | --sidecar-revision)
+            CURSORLESS_SIDECAR_REV="$2"
+            shift
+            shift
+            ;;
+        *)
+            POSITIONAL_ARGS+=("$1")
+            shift
+            break
```

Comment:
How come you break after the first positional arg?

Reply:
No good reason :upside_down_face: